### DISCUSSION TITLE
Does it work for PHP under PHPStorm?

### DISCUSSION THREAD

**Discussion by speller** [community/community#7848] [Copilot Conversations] [answered]
I've installed COdePilot plugin under PHPStorm but nothing happens, no suggestions. Am I doing something wrong?

**Comment by linux-china** [comment, marked as answer]
yes, it works with last PHPStorm 2021.3 Beta

![Snip20211118_32](https://user-images.githubusercontent.com/46711/142480825-61a47a5b-8319-4e7b-898a-36754f6556da.png)

**Reply by jansorg** [reply]
@speller You should get suggestions for PHP. Please post a snippet if you can't make it work.

**Reply by speller** [reply]
@linux-china @jansorg It works now. No idea why it didn't previously. Maybe it was a bug in older versions. Thank you.

**Reply by smileBeda** [reply]
No suggestions here either for PHP. Works for CSS, JS etc.
The same issue is reported by other users, too, so I would suspect there's more to it than "just works", since several people seem to experience problems with PHP files.
